app/services/foreman/plugin.rb

```diff
@@ -396,5 +417,47 @@ def extend_template_helpers_by_module(mod)
       end
       allowed_template_helpers(*(mod.public_instance_methods - Module.public_instance_methods))
     end
+
+    def add_manager_roles(resources, opts)
+      manager = Role.find_by :name => "Manager"
+      return if !manager || resources.empty?
+      add_permissions manager, resources, opts
+    end
```

Comment:
The idea is that all permissions for a given resource are added to 'Manager', regardless of how the permission is named, unless it is explicitly stated they should be skipped by using `:except => ['some_permission_name']` in opts